ec2/sc-ec2-linux-jumpcloud-notebook.yaml

```diff
@@ -388,6 +400,42 @@ Resources:
           Value: "yes"
         - Key: "PatchGroup"
           Value: "prod-default"
+
+  EC2TargetGroup:
+    Type: AWS::ElasticLoadBalancingV2::TargetGroup
+    Properties:
+      Name: !Sub 'TargetGroup-${LinuxInstance}'
```

Comment:
any reason why you need to explicitly name this?  would be preferabe to let CFN name stuff for us.

Reply:
Removed name to avoid replacement of target group when the instance is replaced. Reference: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticloadbalancingv2-targetgroup.html#cfn-elasticloadbalancingv2-targetgroup-name